waku/waku_test.go

```diff
@@ -1650,11 +1710,10 @@ func TestMailserverCompletionEvent(t *testing.T) {
 
 	envelopes := []*common.Envelope{{Data: []byte{1}}, {Data: []byte{2}}}
 	go func() {
-
 		require.NoError(t, peer2.Start())
 		require.NoError(t, p2p.Send(rw2, v0.P2PMessageCode, envelopes))
 		require.NoError(t, p2p.Send(rw2, v0.P2PRequestCompleteCode, [100]byte{})) // 2 hashes + cursor size
-		rw2.Close()
+		require.NoError(t, rw2.Close())
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)